src/libraries/Microsoft.PowerFx.Core/Syntax/Visitors/ListFunctionVisitor.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+
+namespace Microsoft.PowerFx.Syntax
+{
+    internal class ListFunctionVisitor : IdentityTexlVisitor
+    {
+        // FullName --> Name 
+        // Use Fullname as key because it's unique. 
+        public Dictionary<string, string> _functionNames = new Dictionary<string, string>();
```

Comment:
>Dictionary<string, string> [](http://example.com/codeflow?start=15&length=26)

Since the unqualified names are not used, consider changing it to a HashSet<string> instead.